tests/features/interactive_steps.feature

```diff
@@ -101,3 +101,15 @@ Scenario: handles settings from Github secrets
   Then I see the phrase "secret-var2-value"
   When I tap to continue
   And I set the variable "third_text_entry" to secret "SECRET_NOT_THERE"
+
+@i3 @tap-elements @tabs
+Scenario: tap elements
+  Given I start the interview at "test_taps"
+  And I tap the "Tests-first_template-tab" tab
+  Then I see the phrase "Mechanics"
+  And I tap the "Tests-second_template-tab" tab
+  Then I see the phrase "villify"
+  And I tap the "Tests-third_template-tab" tab
+  Then I see the phrase "museum"
+  And I tap the "#special_event" element and wait 5 seconds
```

Comment:
Future issue: Would waiting 1 second be enough? I don't see a need for 5 and our tests are going to continue growing. I'd rather keep the time we add manually to a minimum.

Doesn't need to block this PR.

Reply:
I thought I had a reason for 5, but 1 second seems to work fine locally at least. I changed it to 1 (and added another fix to the step regex to allow for grammatically correct steps), hopefully it's not too flaky.